src/video/kmsdrm/SDL_kmsdrmopengles.c

```diff
@@ -41,7 +41,7 @@ KMSDRM_GLES_DefaultProfileConfig(_THIS, int *mask, int *major, int *minor)
 {
     /* if SDL was _also_ built with the Raspberry Pi driver (so we're
        definitely a Pi device), default to GLES2. */
-#if SDL_VIDEO_DRIVER_RPI
+#if defined(SDL_VIDEO_DRIVER_RPI) || defined(SDL_VIDEO_DRIVER_ROCKCHIP)
```

Comment:
Update the comment too

Reply:
Updating the comment to reflect ROCKCHIP platform. I just don't know what happens to PR if I change that.

Reply:
You can just commit the change to this branch and it will update the PR automatically.